<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelModifierEditor</class>
 <widget class="QDialog" name="ChannelModifierEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel Modifier Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="m_mainGrid">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QSpinBox" name="m_modifiedDMXSpin">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Modified DMX value</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Original DMX value</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="m_origDMXSpin">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QToolButton" name="m_deleteHandlerButton">
         <property name="toolTip">
          <string>Remove the selected handler</string>
         </property>
         <property name="icon">
          <iconset resource="qlcui.qrc">
           <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="m_saveButton">
         <property name="toolTip">
          <string>Overwrite the current template</string>
         </property>
         <property name="icon">
          <iconset resource="qlcui.qrc">
           <normaloff>:/filesave.png</normaloff>:/filesave.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="m_addHandlerButton">
         <property name="toolTip">
          <string>Add a new handler</string>
         </property>
         <property name="icon">
          <iconset resource="qlcui.qrc">
           <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" rowspan="4">
        <widget class="QTreeWidget" name="m_templatesTree">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Templates</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_templateNameEdit">
         <property name="text">
          <string>New Template</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_unsetButton">
       <property name="text">
        <string>Unset Modifier</string>
       </property>
       <property name="icon">
        <iconset resource="qlcui.qrc">
         <normaloff>:/uncheck.png</normaloff>:/uncheck.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qlcui.qrc"/>
  <include location="qlcui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChannelModifierEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChannelModifierEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
